pages/vi/profiles/pattanawadee88.md

```diff
@@ -0,0 +1,8 @@
+# My name: Pattanawadee (Pat)
+**I am an Asia who fell in love with programming.**
+## Location: Boston, US
+### Studying at: Wentworth Institute of Technology
+ [My linkedin](https://www.linkedin.com/in/pattanawadeewinyarat/)
+* OS: Mac OS 10.14.6
```

Comment:
If you are putting bullet here, then it is not working please check https://raw.githack.com/pattanawadee88/pattanawadee88.github.io/my-first-branch/#!pages/vi/profiles/pattanawadee88.md